notebooks/1_BayesianWorkflow/1_Ins_BayesRefresher.ipynb

```diff
@@ -0,0 +1,115 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Section 1: Bayesian refresher and Introduction to ArviZ"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "_What does an end to end Bayesian Workflow look like?_"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Learning Objectives\n",
+    "* Refresh our understanding of Bayes Theorom\n",
+    "* Fit a small binomial model\n",
+    "* Show how a full statistical workflow, even outside of Bayesian Methods, requires more steps more than just model fitting\n",
+    "* Introduce ArviZ"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Bayes Theorem\n",
+    "### The most common formulation\n",
+    "$$\n",
+    "\\Large\n",
+    "P(\\theta | y) = \\frac{ P(y) | \\theta)p(\\theta)}{p(y}\n",
```

Comment:
missing parentheses p(y), I like to use `\mid` instead of `|`, as the former one introduces more space between symbols.